package.json

```diff
@@ -19,6 +19,7 @@
     "mongoose": "^5.10.7"
   },
   "devDependencies": {
+    "morgan": "^1.10.0",
```

Comment:
Oh, I've deleted it today cause I've noticed tghat we dont use it in our docker, do we?

Reply:
When I built the image it broke right away.

Reply:
So I've added this dependency because it was screaming on me for not having it.

Reply:
ah, I see. I uninstalled it but forgot to remove import:)
We actually using it, it was commented out because it's broking the app when you deploy it to Heroku. So I've added check if process isn't production

Reply:
hmm, but this condition break prod anyway. interesting.
`if (isDev) app.use(morgan('dev'))`

Reply:
Huh, interesting